web/packages/design/src/theme/themeColors.story.js

```diff
@@ -289,6 +289,29 @@ const ColorsComponent = () => {
             </Text>
           </Flex>
         </Flex>
+        <Text mb="2" typography="h3">
+          Interactive Colors
+        </Text>
+        <Text mb="2">
+          Interactive colors are used for hover states, indicators, etc. An
+          example of this in use currently would be unified resource cards in
+          the Pinned and Pinned(Hovered) states.
+        </Text>
+        <SingleColorBox
+          mb="2"
+          path="theme.colors.interactive.tonal.primary[0]"
+          color={theme.colors.interactive.tonal.primary[0]}
+        />
+        <SingleColorBox
+          mb="2"
+          path="theme.colors.interactive.tonal.primary[1]"
+          color={theme.colors.interactive.tonal.primary[1]}
+        />
+        <SingleColorBox
+          mb="2"
+          path="theme.colors.interactive.tonal.primary[2]"
+          color={theme.colors.interactive.tonal.primary[2]}
+        />
```

Comment:
can we make this render similar to spot background? (all in one line)
<img width="849" alt="image" src="https://github.com/gravitational/teleport/assets/43280172/4611ddb4-a60a-4be0-aadb-9ebda1f3bf4e">

Reply:
Added but with a slight gap since the title string was overlapping